### ISSUE TITLE
Error: R version - 4.0.0 - was not found on CRAN.

### ISSUE THREAD

**Issue by magnusahlstroem** [ficonsulting/RInno#162]
It seems that when running this code

R_versions <- c(unique(stats::na.omit(stringr::str_extract(readLines("https://cran.rstudio.com/bin/windows/base/", warn = F), "[1-3]\\.[0-9]+\\.[0-9]+"))),
stats::na.omit(stringr::str_extract(readLines("https://cran.rstudio.com/bin/windows/base/old/", warn = F), "[1-3]\\.[0-9]+\\.[0-9]+")))

It only gets R versions up to 3.6.3

\Magnus

**Comment by marcwarmoes** [comment]
Guess it should be:
R_versions <- c(unique(stats::na.omit(stringr::str_extract(readLines("https://cran.rstudio.com/bin/windows/base/", warn = F), "[1-4]\\.[0-9]+\\.[0-9]+"))),
                stats::na.omit(stringr::str_extract(readLines("https://cran.rstudio.com/bin/windows/base/old/", warn = F), "[1-4]\\.[0-9]+\\.[0-9]+")))

Changed [1-3] to [1-4], you could change it to [1-5] if R 5.x.x is anticipated in the next ten years.
When I set  'R_version   = "3.6.3"' using "create_app", I can create the app and also compile it, but the app still doesn't install from .exe because it can not find R 3.6.3....

**Comment by Mathematinho** [comment]
yes, these changes need to be made in both 
trace(RInno::get_R,edit=T) 
and
 trace(RInno::code_section,edit=T)

**Comment by spfeiffe** [comment]
Could someone please make those changes so that the `RInno` package works again?  Thank you!

**Comment by alessio619** [comment]
Any news on this? I still see the issue :)

**Comment by marcwarmoes** [comment]
There is already a pull request for it

**Comment by alessio619** [comment]
Yeah, I saw that it was approved but has not been merged. However, I installed the branch, and was not fully functional, the same correction needs to be done in the `code_section()` functions (lines 32 and 33) and it works. I hope the project is not a zombie, this is a game-changer.

**Comment by bschneidr** [comment]
Yes, you're right about that @alessio619. I was able to get it working in this commit:
https://github.com/bschneidr/RInno/commit/fe8f2cdd6827c3bc214d200e540c29bd05722701
Really hoping this can get fixed in the RInno package itself.

**Comment by jessekps** [comment]
looking at the description file I see:
```
Maintainer: ORPHANED
```
Changing the status to orphaned and updating some docs seems the only update on github since Feb 2019, almost 4 years. I hope someone can take over, it is a useful package.

**Comment by or-sirina** [comment]
> yes, these changes need to be made in both trace(RInno::get_R,edit=T) and trace(RInno::code_section,edit=T)

Thank you so much, this really helps!
